### DISCUSSION TITLE
Pick & Place and BOM csv files for F7 Lightning Plus?

### DISCUSSION THREAD

**Discussion by Jymper** [keirf/greaseweazle#371] [General]
Greetings,
I was interested in attempting to have some GreaseWeazle F7 Lightning Plus boards manufactured. I have minimal experience with ordering boards. I've ordered boards for a few different projects from jlcpcb. 

Perusing the wiki I've downloaded the Assets or Gerbers for several different boards and some of the files do not contain a pick and place file or a BOM file (in csv format). The Test Jig contains both a bom.csv and cpl.csv which make it perfect for using with PCB manufacturers.

Could a bom.csv and cpl.csv be added to the Assets for the F7 Lightning Plus? Or at least the cpl.csv file? 

I started down the path to try to create the bom.csv and cpl.csv myself but I'm WAY out of my depths. I even downloaded a demo of Proteus but the Assets file contains only the gerbers and do not appear to be a full "project".  For me to create the bom and cpl it appears I would need to build and layout a complete board in a PCB design application.

Thank you for the consideration.

As a side note, bom.csv and cpl.csv are not included in the F7 Lightning or F7 Slim either.

Additional thoughts: I was building a JLCPCB assembly order and realized that the F7 Lightning Plus board has components on the top and bottom of the board. It might be helpful to include bom and cpl files for top only assemblies and complete assemblies as options. Ordering a top and bottom assembly adds quite a bit to the cost of a small run of PCBs.

**Comment by keirf** [comment]
Most of these boards were hand assembled by me. I only switched to factory assembly with the V4 model.

**Reply by Jymper** [reply]
Are there any advantages to the F7 Lightning Plus vs the V4 model?
I mistakenly thought the F7 models were newer.

**Reply by keirf** [reply]
The Lightning Plus has High Speed USB and circuitry to generate 5v power from 12v supply. https://github.com/keirf/greaseweazle/wiki/Greaseweazle-Models

Pretty minor stuff. The faster USB is really not needed.

**Reply by Jymper** [reply]
Thank you!
